server/api/impl/proxy/bindings.go

```diff
@@ -89,7 +96,29 @@ func (p *Proxy) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPref
 			continue
 		}
 
-		if !fql.IsTop() {
+		if locPrefix == apps.LocationCommand {
+			b.Location = apps.Location(app.Manifest.AppID)
+			b.Label = string(app.Manifest.AppID)
+		}
+
+		if fql.IsTop() {
+			if locationsUsed[appB.Location] {
+				// TODO Log to the app that there are two bindings on the top level with the same location
```

Comment:
we should be careful logging from `GetBindings`, it'd be a flood of log messages.

Recommend that we remove the TODO, scan for and carefully reconsider any previous logging in `GetBindings`, maybe even add a comment at the top to be careful and not to include logging here without serious consideration for potential volume.

Reply:
@levb

- I changed the existing ones to DEBUG
- removed the comments here
- added a comment above the function saying to avoid excessive logging

Reply:
Actually just removed the existing ones too